cmd/agent/api/agent/agent.go

```diff
@@ -255,3 +257,17 @@ func getTaggerList(w http.ResponseWriter, r *http.Request) {
 	}
 	w.Write(jsonTags)
 }
+
+func setLogLevel(w http.ResponseWriter, r *http.Request) {
+	log.Info("Got a request to change the log level.")
+	r.ParseForm()
+	ll := html.UnescapeString(r.Form.Get("loglevel"))
+	if err := common.SetupLoggerFromConfig(ll); err != nil {
```

Comment:
just checking: is `common.SetupLoggerFromConfig` safe to be called multiple times? (I haven't checked, but Ithink that up to now it has always been called only once at Agent startup, so it may not have been designed to be called more than once)

Reply:
You're right mentioning it so I've checked it again. It looks fine to me, it is only some variables assignation and no unclosed channels creation or whatever which could leads to leak. Also, the good method of `seelog` is already used (`ReplaceLogger` instead of `UseLogger`).